src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -150,6 +150,13 @@ def make_artifacts(
     "queue. The maximum supported runtime is 3 days. Keep in mind that "
     "runtimes by node vary wildly. ",
 )
+@click.option(
```

Comment:
Move this arg to right after --output-dir.
(1) it makes sense to be next to the output directory stuff
(2) it should definitely come before the --jobmon arg

Reply:
That makes sense.